stackit/internal/services/loadbalancer/loadbalancer_acc_test.go

```diff
@@ -0,0 +1,322 @@
+package loadbalancer_test
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/hashicorp/terraform-plugin-testing/terraform"
+
+	"github.com/stackitcloud/stackit-sdk-go/core/config"
+	"github.com/stackitcloud/stackit-sdk-go/services/loadbalancer"
+	"github.com/stackitcloud/terraform-provider-stackit/stackit/internal/testutil"
+)
+
+// Instance resource data
+var loadBalancerResource = map[string]string{
+	"project_id":            testutil.ProjectId,
+	"name":                  fmt.Sprintf("tf-acc-%s", acctest.RandStringFromCharSet(7, acctest.CharSetAlphaNum)),
+	"target_pool_name":      "example-target-pool",
+	"target_port":           "5432",
+	"target_port_updated":   "5431",
+	"target_display_name":   "example-target",
+	"healthy_threshold":     "3",
+	"interval":              "10s",
+	"interval_jitter":       "5s",
+	"timeout":               "10s",
+	"unhealthy_threshold":   "3",
+	"listener_display_name": "example-listener",
+	"listener_port":         "5432",
+	"listener_protocol":     "PROTOCOL_TCP",
+	"network_role":          "ROLE_LISTENERS_AND_TARGETS",
+	"private_network_only":  "true",
+}
+
+func configResources(targetPort string) string {
+	return fmt.Sprintf(`
+		%s
+
+		%s
+
+		resource "stackit_loadbalancer" "loadbalancer" {
+			project_id = "%s"
+			name       = "%s"
+			target_pools = [
+				{
+				name        = "%s"
+				target_port = %s
+				targets = [
+					{
+					display_name = "%s"
+					ip           = openstack_compute_instance_v2.example.network.0.fixed_ip_v4
+					}
+				]
+				active_health_check = {
+					healthy_threshold   = %s
+					interval            = "%s"
+					interval_jitter     = "%s"
+					timeout             = "%s"
+					unhealthy_threshold = %s
+				}
+				}
+			]
+			listeners = [
+				{
+				display_name = "%s"
+				port         = %s
+				protocol     = "%s"
+				target_pool  = "%s"
+				}
+			]
+			networks = [
+				{
+				network_id = openstack_networking_network_v2.example.id
+				role       = "%s"
+				}
+			]
+			options = {
+				private_network_only = %s
+			}
+			}
+		`,
+		supportingInfraResources(loadBalancerResource["name"], OpenStack{
+			userDomainName: "portal_mvp",
```

Comment:
remove hardcoded values, read from utils

Reply:
Oops, I have deleted those credentials and will create new ones also